### PR TITLE
added tests for Richtext

### PR DESCRIPTION
**Pull request by rrieger-espirit**
In this Ticket I wrote tests for the Rich-text-component

### CODE REVIEW

File: src/components/RichText/__tests__/RichText.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+describe("components/RichText", () => {
+  it("checks if the content will be passed in", () => {
+    const content = "test";
+    const { getByTestId } = render(RichText, {
+      props: {
+        content: content,
+      },
+    });
+    const contentTest = getByTestId("richtext-content");
+    expect(contentTest).toBeTruthy;
```
Comment (tgolbs) [inline src/components/RichText/__tests__/RichText.spec.tsx:13]: Please check for correct content.

Comment (tgolbs) [reply src/components/RichText/__tests__/RichText.spec.tsx:13]: And call toBeTruthy

**Review by tgolbs — CHANGES_REQUESTED** [review]

File: src/components/RichText/__tests__/RichText.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  });
+  it("checks if inline works", () => {
+    const content = "test";
+    const inline = true;
+    const { getByTestId } = render(RichText, {
+      props: {
+        content: content,
+        inline: inline,
+      },
```
Comment (ninoNato) [inline src/components/RichText/__tests__/RichText.spec.tsx:22]: Please use the object initializer shorthand:
`props: { content, inline },`

File: src/components/RichText/__tests__/RichText.spec.tsx (hunk 1 above)
Comment (rrieger-espirit) [reply src/components/RichText/__tests__/RichText.spec.tsx:13]: done

File: src/components/RichText/__tests__/RichText.spec.tsx (hunk 2 above)
Comment (rrieger-espirit) [reply src/components/RichText/__tests__/RichText.spec.tsx]: done

File: src/components/RichText/__tests__/RichText.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { render } from "@testing-library/vue";
+import RichText from "./../";
+
+describe("components/RichText", () => {
+  it("checks if the content will be passed in", () => {
+    const content = "test";
+    const { getByText } = render(RichText, {
+      props: {
+        content: content,
+      },
```
[the remark this answers is not in the document]
Comment (rrieger-espirit) [reply src/components/RichText/__tests__/RichText.spec.tsx:10]: done

**Pull request merged**